packages/completer-extension/schema/tracker.json

```diff
@@ -20,10 +20,22 @@
       }
     },
     "providerTimeout": {
-      "title": "Default timeout for a provider",
+      "title": "Default timeout for a provider.",
       "description": "If a provider can not return the response for a completer request before timeout, the result of this provider will be ignored. Value is in millisecond",
       "type": "number",
       "default": 1000
+    },
+    "showDocumentPanel": {
+      "title": "Show the document panel.",
+      "description": "Show or hide the document panel of completer widget, this configuration is applied to all providers. If a provider does not provide the document content, the panel will be hidden for this provider.",
+      "type": "boolean",
+      "default": true
```

Comment:
Not sure if we want it default to true. Would be good to bring up on dev meeting, but it would probably be fine to merge as default and flip to `false` just before 4.0 release if we find it lacks consensus for enabling it as new default :).